refinery/templates/admin/core/sitestatistics/change_list.html

```diff
@@ -0,0 +1,21 @@
+{% extends "admin/change_list.html" %}
+{% load i18n admin_urls %}
+
+{% block extrahead %}
+  {{ block.super }}
+  <script type="module">
+    // Load the Observable runtime and inspector.
+    import {Runtime, Inspector} from "https://unpkg.com/@observablehq/notebook-runtime?module";
+
+    import notebook from "https://api.observablehq.com/@scottx611x/refinery-stats.js";
```

Comment:
Shouldn't this live in something like https://api.observablehq.com/@refinery-platofrm instead?

Reply:
@hackdna I don't believe it should. That would require us to create another github user: `@refinery-platform`. I'd rather have the link tied to a real person than an umbrella user account.

Reply:
So, what would happen if we lost access to https://api.observablehq.com/@scottx611x and needed to make changes to refinery-stats.js?

Reply:
@hackdna I mean, you could say the same about the refinery-platform github user you are suggesting... Who would have access/administrate that account? Would we be sharing credentials amongst ourselves? If changes really need to be made, the Observable is public and can be forked.

Reply:
Is there no way to provide access to refinery-platform organization?

Reply:
No, at least not at the moment.

https://beta.observablehq.com/terms-of-service

```
1. Required Information
You must log in with a valid GitHub account, linked to a valid email address in order to complete the signup process. Any other information requested, such as your real name, is optional.
```